dev/create-release/release-util.sh

```diff
@@ -21,6 +21,7 @@ DRY_RUN=${DRY_RUN:-0}
 GPG="gpg --no-tty --batch"
 ASF_REPO="https://gitbox.apache.org/repos/asf/spark.git"
 ASF_REPO_WEBUI="https://gitbox.apache.org/repos/asf?p=spark.git"
+ASF_GITHUB_REPO="https://github.com/apache/spark"
```

Comment:
Hm, I wonder if I completely agree with it. Mixing gitbox and github could look a bit confusing. The previous `! curl -s --fail "$ASF_REPO_WEBUI;a=commit;h=$1" | grep '404 Not Found' > /dev/null` is good enough I guess. I'm okay but let's don't do this if it's not only me feeling in this way.

Reply:
It's good enough but it's a hack to me. Using Github repo at this single place is also a hack for now, but according to https://github.com/apache/spark/pull/23476#discussion_r245574264 , seems we can switch to Github repo completely in the future.

Reply:
If we're going to switch to github, I'm okie. Don't block by me on this. Either way is fine to me.

Reply:
Either should be fine. It has just occurred to me though, what if some of us pushed to gitbox and some to github? how would sync resolve it? what rebases on what? I guess it is better to push to one consistently, and I presume most of us (all?) are pushing to github?

Reply:
For now, I don't see a clear LGTM for this PR.
And, at least, I'm currently pushing GitBox only.
Please let me know if the decision is made.

Reply:
Hm, I suppose I think we should consistently push to github. That was the advice I think we sent (?) to committers, and I can update the committer docs on this if any exist. At least we'd have a standard.

Reply:
That sync has been resolved, right? or at least, we do need that to work going forward, rather than work around it. Is it a blocker right now?

Overall I think we should standardize on github.com remotes over gitbox, so I think this could change again to use all the github.com remotes. Is that OK with you? that would be OK with me.